### PR TITLE
Improvements on main page

### PR DESCRIPTION
**Pull request by Vegasq**
- Markers on buildings
- Popupus on click
- Display claims on click

### CODE REVIEW

**Comment by autogestion** [comment]
Thank you for contributing. We decided not to use markers, but show on polygon number of claims, and after the click on number, show the list of claims. Numbers are inserted in geojson, and are available by key ''claim_count'

**Comment by Vegasq** [comment]
> show on polygon number of claims

Cool. Do you have some mockup with example?

File: corruption_tracker/settings.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 
+SECRET_KEY = '8lu*6g0lg)9z!ba+a$ehk)xt)x%rxgb$i1&amp;022shmi1jcgihb*'
```
Comment (autogestion) [inline corruption_tracker/settings.py]: This must not be public, please use local_settings.py

File: corruption_tracker/settings.py [hunk 2]
```diff
@@ -24,6 +26,12 @@
 
 ALLOWED_HOSTS = []
 
+DATABASES = {
```
Comment (autogestion) [inline corruption_tracker/settings.py]: This must not be public, please use local_settings.py

**Comment by Vegasq** [comment]
https://www.dropbox.com/s/o6a3kmst0r15ilj/Screenshot%202015-09-16%2017.56.09.png?dl=0

There is a posibility to replace marker image with div with custom style and custom html inside. Please review screenshot above, probably it is correct direction.

**Comment by Vegasq** [comment]
Oh, crap, settings...

**Comment by autogestion** [comment]
We do not have any design/designer, so no mockups (

**Pull request merged**